docs/google/engine.md

```diff
@@ -273,4 +273,46 @@ historical_results = historical_job.results()
 ```
 
 If you did not save the ids, you can still find them from your
-job using the [Cloud Console](https://console.cloud.google.com/quantum/jobs).
+job using the [Cloud Console](https://console.cloud.google.com/quantum/jobs) or
+by using `cirq.google.Engine.list_jobs()` or `cirq.google.Engine.list_programs()`.
```

Comment:
It might be worth reiterating the difference between jobs or programs here just so people are clear which one to use.

Reply:
Sounds good - I broke out the two sections separately and added a bit more explanation.